iroh-one/src/main.rs

```diff
@@ -35,6 +36,14 @@ async fn main() -> Result<()> {
     )
     .unwrap();
 
+    #[cfg(unix)]
+    {
+        match iroh_util::increase_fd_limit() {
```

Comment:
@dignifiedquire I'm learning rust now and curious about code duplication here. Since there are three identical pieces of calls to `iroh_util::increase_fd_limit()` with `match` and error checks. Would it be frowned upon to move this error check into `iroh_util::increase_fd_limit()` instead? (also the `use tracing::{debug, error}` can be moved).

Reply:
potentially, I m a bit torn on it here as the usages are different binaries. but agree with the sentiment

Reply:
I mostly did it because error handling might be different in different use cases even though they are the same now 🤷